Library/Homebrew/formula_installer.rb

```diff
@@ -863,9 +864,13 @@ def link(keg)
   end
 
   def install_plist
-    return unless formula.plist
+    return unless plist = formula.plist
```

Comment:
Not sure I understand this bit; why use `formula.plist` vs. `plist` below?

Reply:
Maybe there is a more proper way I am missing, but for some reason trying to reassign `formula.plist =` just throws a "`plist=` undefined method" error… doing it this way gets around that.